subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/EmulatorSet.kt

```diff
@@ -5,12 +5,12 @@ import com.avito.instrumentation.reservation.request.Device
 object EmulatorSet {
     val fast = setOf(
         Device.Emulator.Emulator22,
-        Device.Emulator.Emulator27
+        Device.Emulator.Emulator28
```

Comment:
Help me, why did we decide to replace `27 API`?

Reply:
Because it's a set for the fast check. It should be rapid and limited to 40 minutes. So we can run our tests only on 2 emulators. It seems consequent to run tests on the minimum supported API and the target supported API.

Reply:
Not actual. I've removed `Emulator28` from these sets.